components/Profile/pack-list.tsx

```diff
@@ -46,18 +45,18 @@ export function PackList({ type, id, canEdit }: PackListProps) {
 	}, [type, id])
 
 	const sortedPacks = [...packs].sort((a, b) => {
-		if (a.featured && !b.featured) return -1
-		if (!a.featured && b.featured) return 1
+		if (a.popular && !b.popular) return -1
+		if (!a.popular && b.popular) return 1
 		return 0
 	})
 
 	const groupedPacks = sortedPacks.reduce(
 		(acc, pack) => {
 			if (type === 'created') {
-				if (pack.pending) {
+				if (pack.status === 'pending') {
 					acc.pending.push(pack)
-				} else if (pack.denied) {
-					acc.denied.push(pack)
+				} else if (pack.status === 'resubmit_pending') {
```

Comment:
This should also use the enum
```suggestion
				} else if (pack.status === Status.resubmit_pending) {
```